File: day7/phoneBook/PhoneBook.java
import java.util.Scanner;
import java.util.ArrayList;
public class PhoneBook{

public static ArrayList<Contact> contacts = new ArrayList<>();
static Scanner scanner = new Scanner(System.in);

public static void main(String[] args){

Contact phone = addContact();
System.out.print(phone);

}

public static Contact addContact(){
	System.out.println("Enter first name: ");
	String firstName = scanner.nextLine();
	System.out.println("Enter last name: ");
	String lastName = scanner.nextLine();
	System.out.println("Enterphone number");
	String number = scanner.nextLine();
	Contact contact = new Contact(firstName, lastName, number ); 
	contacts.add(contact);
	return contact;
	

}

public static String removeContact(){
	System.out.print("Enter contact to delete: ");
	}
	


}

File: day7/phoneBook/phoneBook.py

contacts = []

def add_contact():
  """
  Prompts the user for contact information and creates a new Contact object.
  """
  print("Enter first name: ")
  first_name = input()
  print("Enter last name: ")
  last_name = input()
  print("Enter phone number: ")
  phone_number = input()
  contact = Contact(first_name, last_name, phone_number)
  contacts.append(contact)
  return contact

def remove_contact():
  """
  Prompts the user for a contact name to delete and removes it from the list.
  """
  print("Enter contact to delete: ")
  name_to_delete = input()
  for i, contact in enumerate(contacts):
    if contact.first_name == name_to_delete or contact.last_name == name_to_delete:
      del contacts[i]
      print("Contact deleted")
      return